### DISCUSSION TITLE
How to Join Slack

### DISCUSSION THREAD

**Discussion by nickmcsimpson** [openfoodfacts/openfoodfacts-server#5187] [Q&A] [answered]
I recently stumbled upon this set of projects in search of documentation around product packaging and want to contribute. I've seen multiple links to the Slack channel, but it seems one cannot freely join. **I am wondering how to get the invite.**

Spending time on the platform getting a feel for how things work. I hope to contribute to the Eco-score project.

Thanks!

**Comment by nickmcsimpson** [comment]
Found the proper link in the application once I started running it locally. The Readme and other links should be updated.

**Comment by VaiTon** [comment, marked as answer]
The official and updated link is: https://slack.openfoodfacts.org/
